uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/build.js

```diff
@@ -5,6 +5,7 @@
   modules: [
     {name: 'admin-dashboard'},
     {name: 'absence-tab'},
+    {name: 'calendar-feeds.list'},
```

Comment:
Could you please add spaces to objects: `{ name: 'calendar-feeds.list' }` + other objects?

The semistandard will still complain on the "expected expression" issue (or smth like that), you may then use `--no-verify` flag to bypass as an exception.